test/e2e/specs/specs-wpcom/wp-calypso-gutenberg-site-editor-tracking-spec.js

```diff
@@ -1,6 +1,12 @@
+/**
+ * External dependencies
+ */
```

Comment:
FYI, it appears we don't need these import comments any longer. More info: p4TIVU-9O9-p2

Reply:
🤔   The linter forced me to do this after I rebased on Monday, there was no issue before that.

Reply:
😕

cc @scinos

Reply:
I can delete those comments in `trunk` and eslint doesn't seem to complain to me.

Also all code style checks for this branch have been always green (But maybe some builds are missing?): 
![image](https://user-images.githubusercontent.com/975703/131731916-d2fa35bf-a303-4bd8-9366-42442f8cd219.png)

Maybe it was a polluted `node_modules` that still had the old rule active or something like that.

Reply:
> Maybe it was a polluted node_modules that still had the old rule active or something like that.

Seems to be!  After cleaning and reinstalling the node modules the linter isn't mad at me in these files.  Odd the linter never said anything until Tuesday though?  Oh well. 😆